Makefile

```diff
@@ -166,7 +166,16 @@ local-install-chrome:
 	install -m 0755 linux-entra-sso.py ~/.config/google-chrome
 	${Q}sed -i $(UPDATE_VERSION_PY) ~/.config/google-chrome/linux-entra-sso.py
 
-local-install: local-install-firefox local-install-chrome
+local-install-vivaldi:
+	install -d ~/.config/vivaldi/NativeMessagingHosts
+	install -m 0644 platform/chrome/linux_entra_sso.json ~/.config/vivaldi/NativeMessagingHosts
+	${Q}sed -i 's|/usr/local/lib/linux-entra-sso/|'$(HOME)'/.config/vivaldi/|' ~/.config/vivaldi/NativeMessagingHosts/linux_entra_sso.json
+	# compute extension id and and grant permission
+	${Q}sed -i 's|{extension_id}|$(CHROME_EXT_ID)|' ~/.config/vivaldi/NativeMessagingHosts/linux_entra_sso.json
```

Comment:
Hi, thanks for this contribution. AFAIK the Vivaldi browser is chromium based. Is there a particular reason to have a dedicated install target for that? Currently both chrome and chromium are installed in the same target ("local-install-chrome"). To keep things consistent, we probably better move this part to the shared chrome install target. Opinions? @jan-kiszka

Reply:
@fmoessbauer For some reason it didn't work for me initially. I tested again now from scratch and doing "local-install-chrome" indeed makes it work in Vivaldi also.
Sorry for the confusion, I will close this PR.

Thank you!

Reply:
Are you sure that the `platform/chrome/` or `platform/chromium` paths are sufficient? According to https://forum.vivaldi.net/post/382945 the path is actually `~/.config/vivaldi/NativeMessagingHosts`. I was just referring to adding this path to the `local-install-chrome` target (e.g. below chromium) instead of defining its own target.

Reply:
Yeah I saw that forum post, thats how I started. It is working now with only local-install-chrome, and I don't have anything installed in ~/.config/vivaldi
```shell
find ~/.config/vivaldi -name '*linux[-_]entra[-_]sso*'
/home/kbcz/.config/vivaldi/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_48.png
/home/kbcz/.config/vivaldi/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_128.png.license
/home/kbcz/.config/vivaldi/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_128.png
/home/kbcz/.config/vivaldi/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_48.png.license
```

```shell
find ~/.config/google-chrome -name '*linux[-_]entra[-_]sso*'
/home/kbcz/.config/google-chrome/linux-entra-sso.py
/home/kbcz/.config/google-chrome/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_48.png
/home/kbcz/.config/google-chrome/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_128.png.license
/home/kbcz/.config/google-chrome/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_128.png
/home/kbcz/.config/google-chrome/Default/Extensions/jlnfnnolkbjieggibinobhkjdfbpcohn/1.1.0_0/icons/linux-entra-sso_48.png.license
/home/kbcz/.config/google-chrome/NativeMessagingHosts/linux_entra_sso.json
```

is there anything else I could check?

Thanks again for looking into this.

Reply:
Additionally, I was thinking of just adding a path under local-install-chrome but it seemed too invasive for users who do not have Vivaldi installed.

Reply:
> is there anything else I could check?

As of now, I don't think so. But feel free to come back to us in case it is not properly working in Vivaldi. Then we can re-consider this MR.

> Additionally, I was thinking of just adding a path under local-install-chrome but it seemed too invasive for users who do not have Vivaldi installed.

Well... I have no problem with that as its just a bunch of config files. If things are packaged by distros, they anyways decide by themselves what to include and how to split things. I just want to keep things consistent within this project / Makefile.